src/frontend/components/DashboardDocumentTitleForm/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { Button, Form, FormField, Text, TextInput } from 'grommet';
+import React, { Fragment, useState } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { updateResource } from '../../data/sideEffects/updateResource';
+import { useDocument } from '../../data/stores/useDocument';
+import { Document } from '../../types/file';
+import { modelName } from '../../types/models';
+import { Maybe } from '../../utils/types';
+
+const messages = defineMessages({
+  updateError: {
+    defaultMessage: 'Impossible to update the title. Try again later.',
```

Comment:
I think we need different error message for `400` error for 0 character in the field and more than the limit (happens to be 255)

Reply:
We discuss about that with @sampaccoud. I totally agree with you, we should have the failure reason. But DRF just return a string and we can't rely on it. This string can change so our traduction will break and we will don't see it. What I can do is to use all the HTML5 input validation parameter like `maxlength` and `required`. WDYT ?

Reply:
That sounds great! if we already now about these errors we don't need to wait for the server to tell us

Reply:
@mbenadda the only thing better would be to make sure that the limit on the server will stay coherent with the limit on the frontend. Any idea how to do this?

Reply:
It would need to be available in the template or as an API endpoint I guess.

Or we could start up a JSON file shared by Django settings and React if we think this is becoming a regular need?

Reply:
For now I think it's overkill to do that.